parquet/src/basic.rs

```diff
@@ -304,6 +316,18 @@ impl FromStr for Encoding {
 // Mirrors `parquet::CompressionCodec`
 
 /// Supported compression algorithms.
```

Comment:
```suggestion
/// Supported block compression algorithms. Defaults to ``Self::UNCOMPRESSED``
```

If we are changing the docs anyway, maybe we can also call out that the default compression is `NONE` as I at least have been surprised by that in the past